standard-controller/src/main/java/io/enmasse/controller/standard/RouterStatus.java

```diff
@@ -83,26 +91,37 @@ public int checkLinkRoutes(Address address) {
 
     public static int checkActiveAutoLink(Address address, List<RouterStatus> routerStatusList) {
         int ok = 0;
-        Set<String> active = new HashSet<>();
+        final Set<String> active = new HashSet<>();
+        final String addressName = address.getSpec().getAddress();
+
+        log.debug("Address: {}", addressName);
 
         for (RouterStatus routerStatus : routerStatusList) {
 
+            log.debug("Router Auto Links: {}", routerStatus.autoLinks);
+
             for (List<String> autoLink : routerStatus.autoLinks) {
                 if (autoLink.size() > 3) {
                     String addr = autoLink.get(0);
                     String dir = autoLink.get(2);
                     String operStatus = autoLink.get(3);
 
-                    if (addr.equals(address.getSpec().getAddress()) && operStatus.equals("active")) {
+                    log.debug("Addr: {}, Dir: {}, operStatus: {}", addr, dir, operStatus);
+
+                    if (addr.equals(addressName) && operStatus.equals("active")) {
                         active.add(dir);
+                        log.debug("  Match!");
                     }
                 }
             }
         }
 
         if (active.size() < 2) {
-            address.getStatus().setReady(false).appendMessage("Address " + address.getSpec().getAddress() + " is missing active autoLink (active in dirs: " + active + ")");
+            var msg = "Address " + addressName + " is missing active autoLink (active in dirs: " + active + ")";
+            log.info(msg);
```

Comment:
This would be a bit verbose with many addresses, maybe change it to debug.

Reply:
Changed that to "debug"